tests/shared/core/training_data/story_writer/test_markdown_story_writer.py

```diff
@@ -0,0 +1,20 @@
+from rasa.core.agent import Agent
+from rasa.shared.core.training_data.story_writer.markdown_story_writer import (
+    MarkdownStoryWriter,
+)
+
+
+async def test_tracker_dump_e2e_story(default_agent: Agent):
```

Comment:
Do we really need this test? MD is deprecated now.

Reply:
@joejuzl could you please take this over

Reply:
I didn't find any test for writing a conversation test using the markdown writer. We still have to maintain this until 3.0.0 so it seemed a test would help.